main.go

```diff
@@ -2,9 +2,14 @@ package main
 
 import (
 	"os"
+	"fmt"
 )
 
+var vetError = fmt.Sprintf("", 1, 2, 3)
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
no formatting directive in Sprintf call